modules/core/src/test/scala/org/scalasteward/core/edit/UpdateHeuristicTest.scala

```diff
@@ -243,6 +243,26 @@ class UpdateHeuristicTest extends FunSuite with Matchers {
       .replaceVersionIn(original) shouldBe (Some(expected) -> UpdateHeuristic.groupId.name)
   }
 
+  test("specific to scalafmt: should be Scala version agnostic") {
```

Comment:
Actually, those tests are needed for this fix.

Added just to ensure that whitespace does not matter https://github.com/fthomas/scala-steward/pull/842#issuecomment-522227235